plugins/ssh/current_ssh.ml

```diff
@@ -0,0 +1,8 @@
+open Current.Syntax
+
+module R = Current_cache.Output(Run)
+
+let run ~schedule ~key host args =
+  Current.component "ssh@,%s" host |>
+  let> args = args in
+  R.set ~schedule host key { Run.Value.args }
```

Comment:
I might need clarification, but shouldn't the key be derived from the args? How is the key supposed to change?

Reply:
Definitely not, because then you couldn't run the same command twice.  You pass the key when you invoke the call `Current_ssh.run ... ~key:"my-key" ... `.  You may want to run `ls` twice, perhaps once before and once after some other commands.  The first would be called with `~key:"ls1"` and the second as `~key:"ls2"`.

Reply:
Thanks for the clarification! That's clearer. My concern is that the `key` word in this way seems to be tight to `OCurrent`. I would use something like `label` instead. However, this is opinionated, so feel free to ignore it :)

Reply:
I went with `key` to match the definition in `lib_cache/s.ml`